src/runtimes/node/index.ts

```diff
@@ -151,9 +151,9 @@ const zipFunction: ZipFunction = async function ({
     bundler: bundlerName,
     bundlerWarnings,
     config,
-    displayName: config?.name,
+    displayName: config?.name ?? staticAnalysisName,
```

Comment:
Should the ISC value not take precedence?

Reply:
Both is fine. I gave precedence to the existing value, because that way it's not a breaking change for functions that have both defined right now. Happy to flip though.

Reply:
yes, looks like we're giving ISC precedence for edge functions: https://github.com/netlify/edge-bundler/blob/0af1086fb92fa4eaa3e63709baac40df3f9ce06c/packages/edge-bundler/node/declaration.test.ts#L32

Reply:
flipped precedence in https://github.com/netlify/zip-it-and-ship-it/pull/1732/commits/14ac467e6b4ae3cff7f805cd01e432d19ea3d4d6